lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -442,14 +440,17 @@ protected function _getPortableTableColumnDefinition($tableColumn)
             'default'       => $tableColumn['default'],
             'precision'     => $precision,
             'scale'         => $scale,
-            'fixed'         => $fixed,
             'unsigned'      => false,
             'autoincrement' => $autoincrement,
             'comment'       => isset($tableColumn['comment']) && $tableColumn['comment'] !== ''
                 ? $tableColumn['comment']
                 : null,
         ];
 
+        if ($fixed !== null) {
```

Comment:
What's the background of this change?

Reply:
`null` is not a valid value of a boolean parameter but it's initialized with a `null` and then assigned a boolean value based on the column metadata. We pass the value down to the column definition only if it was explicitly assigned.